test/cppsuite/test_harness/configuration_settings.h

```diff
@@ -45,7 +42,7 @@ class configuration {
     std::string
     get_config()
     {
-        return _config;
+        return (_config);
```

Comment:
So for the c++ coding style, I see we still return in brackets. Is this a decision made?

Reply:
This is a question I'll bring up in the Wednesday meeting, I think we'll have to have a wiki page defining the WiredTiger C++ style. Or opt to use the WiredTiger C style. For now lets just stick with the brackets until we've had that meeting. Currently our other C++ code, Workgen and Xray have varied style between them.